{"message":"undefined, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-12 21:45:12"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-12 21:45:12"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-12 21:45:12"}
{"message":"undefined, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-12 21:46:29"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-12 21:46:29"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-12 21:46:29"}
{"message":"undefined, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-12 21:47:38"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-12 21:47:38"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-12 21:47:38"}
{"message":"undefined, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-12 21:48:07"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-12 21:48:07"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-12 21:48:07"}
{"message":"undefined, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-12 21:48:37"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-12 21:48:37"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-12 21:48:37"}
{"message":"production, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-12 21:51:17"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-12 21:51:17"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-12 21:51:17"}
